src/util.ts

```diff
@@ -648,9 +648,7 @@ export function isArrayOfString(x: any): x is string[] {
 }
 
 export function isNullOrUndefined(x?: any): boolean {
-  // Double equals provides the correct answer for 'null' and 'undefined'
-  // http://www.ecma-international.org/ecma-262/6.0/index.html#sec-abstract-equality-comparison
-  return x === null;
+  return (x === null || x === undefined);
```

Comment:
Ah.  It looks like this broke recently.  It used to be `x == null`.  I guess to keep the linter happy, this new approach is better.